packages/eslint-config-eslint/default.yml

```diff
@@ -4,6 +4,10 @@ extends:
     - "plugin:node/recommended"
     - "plugin:jsdoc/recommended"
     - "plugin:eslint-comments/recommended"
+    # - "plugin:sonarjs/recommended"
```

Comment:
Commented out? Why did we stop using the config?

Reply:
Had been intended as a reminder or to provoke a comment such as yours so we could add it. :-)